tests/shared/src/test/scala-3/scala/meta/tests/quasiquotes/Scala3SpecificSuccessSuite.scala

```diff
@@ -0,0 +1,24 @@
+package scala.meta.tests
+
+package quasiquotes
+
+import scala.meta._
+import scala.meta.dialects.Scala3
+
+class Scala3SpecificSuccessSuite extends TreeSuiteBase {
+
+  test("multiline - parse when inlined") {
```

Comment:
where's the failed assert here? also, when the fix is added in the second commit, why doesn't this test change, to show improvement?